pkg/providers/aws.go

```diff
@@ -35,18 +35,18 @@ var _ = Describe("MetadataServiceOptions", framework.LabelCloudProviderSpecific,
 	BeforeEach(func() {
 		var err error
 		client, err = framework.LoadClient()
-		Expect(err).ToNot(HaveOccurred())
+		Expect(err).ToNot(HaveOccurred(), "Failed to load client")
```

Comment:
Should annotations like these be made more clear on which component they're failing to load, or is leaving them like this fine?

Reply:
as long as it is clear where the error originates from, and is not ambiguous with other error messages, then it's fine for me.